Graphs/Table/index.js

```diff
@@ -401,7 +401,7 @@ const TableGraph = (props) => {
     }
 
     const onScroll = ({ startIndex }) => {
-        const page = (startIndex / LIMIT) + 1;
+        const page = (startIndex / (scrollData.pageSize || LIMIT)) + 1;
```

Comment:
1. use empty object `{}` as default value for scrollData
2. change check here using isVariableNonNullAndDefined,  `const pageSize = isVariableNonNullAndDefined(scrollData.pageSize) ? scrollData.pageSize : LIMIT;`